### ISSUE TITLE
Cannot check security.json from shopware-static-data

### ISSUE THREAD

**Issue by hesse-wct** [FriendsOfShopware/shopware-static-data#14]
### Question

![Image](https://github.com/user-attachments/assets/25869f61-a657-42bb-9bd7-d17b4cffcb91)

What exactly am I supposed to do about this message? 
Shopware Version 6.4.1.8., Tools Version 0.2.20.

Never had this error before and the links just gives me the .json, but where do I need it. Thank you. :D

**Comment by shyim** [comment]
does the error persist?

**Comment by hesse-wct** [comment]
Yes, having it for a couple of days now. Tried to get rid off it today but I could not find something helpful what I'm supposed to do.

**Comment by tinect** [comment]
Which plugin version do you have installed?
